src/Validation/Validation.ts

```diff
@@ -0,0 +1,20 @@
+export const PasswordValidation = (minLength: number = 8) => ({
+  required: "Password is required",
+  minLength: {
+    value: minLength,
+    message: `Password must be at least ${minLength} characters long`,
+  },
+  validate: {
```

Comment:
This is better than including all of the validations in one regular expression. This will allow us to display one validation pattern at a time instead of throwing multiple requirements at once.